resultbrowser/src/css/default.css

```diff
@@ -594,4 +609,4 @@ table.key-value-table td.value .csep {
 
 .no-wrap {
     white-space: nowrap;
-}
+}
```

Comment:
Please do NOT remove the final newline at end-of-file.

Reply:
Done